.circleci/verbatim-sources/commands.yml

```diff
@@ -143,10 +143,6 @@ commands:
         type: string
         default: ""
     steps:
-      - run:
```

Comment:
This actually isn't necessary, checked on the executor and it defaults to python 3.8.x for `pip3`